paper_framework_sources/p2079_system_execution_context.bs

```diff
@@ -58,79 +65,171 @@ This was removed from [[P2300R9]] based on that and other input.
 
+sender auto begin = schedule(sch);
+sender auto hi = then(begin, []{
+    std::cout << "Hello world! Have an int.";
+    return 13;
+});
+sender auto add_42 = then(hi, [](int arg) { return arg + 42; });
+
+auto [i] = this_thread::sync_wait(add_42).value();
+```
+
+We can structure the same thing using `execution::on`, which better matches structured concurrency:
 ```cpp
-class system_context {
-public:
-  system_context();
-   ~system_context();
+using namespace std::execution;
 
-  system_context(const system_context&) = delete;
-  system_context(system_context&&) = delete;
-  system_context& operator=(const system_context&) = delete;
-  system_context& operator=(system_context&&) = delete;
+system_context ctx;
+scheduler auto sch = ctx.scheduler();
 
-  implementation-defined-system_scheduler get_scheduler();
-  size_t max_concurrency() const noexcept;
-};
+sender auto hi = then(just(), []{
+    std::cout << "Hello world! Have an int.";
+    return 13;
+});
+sender auto add_42 = then(hi, [](int arg) { return arg + 42; });
+
+auto [i] = this_thread::sync_wait(on(sch, add_42)).value();
 ```
 
- - On construction, the `system_context` may initialize a shared system context, if it has not been previously initialized.
- - To support sharing of an underlying system context, two `system_context` objects do not guarantee task isolation.
-     If work submitted by one can consume the thread pool, that can block progress of another.
- - The `system_context` is non-copyable and non-moveable.
- - The `system_context` must outlive work launched on it.
-     If there is outstanding work at the point of destruction, `std::terminate` will be called.
- - The `system_context` must outlive schedulers obtained from it.
-     If there are outstanding schedulers at destruction time, this is undefined behavior.
- - The lifetime of a `system_context` must be fully contained in the lifetime of `main()`.
- - `get_scheduler` returns a `system_scheduler` instance that holds a reference to the `system_context`.
- - `max_concurrency` will return a value representing the maximum number of threads the context may support.
-     This is not a snapshot of the current number of threads, and may return `numeric_limits<size_t>::max`.
-
-## `system_scheduler` ## {#system_scheduler}
-
-A `system_scheduler` is a copyable handle to a `system_context`. It is the means through which agents are launched on a `system_context`.
-The `system_scheduler` instance does not have to outlive work submitted to it.
-The `system_scheduler` is technically implementation-defined, but must be nameable.
-See later discussion for how this might work.
+The `system_scheduler` customises `bulk`, so we can use `bulk` dependent on the scheduler.
+Here we use it in structured form using the parameterless `get_scheduler` that retrieves the scheduler from the receiver, combined with `on`:
+```cpp
+auto bar() {
+  return
+    ex::let_value(
+      ex::get_scheduler(),          // Fetch scheduler from receiver.
```

Comment:
Seems like this is outdated usage or get_scheduler() according to what we discussed on the sync up